subprojects/xcfa/xcfa/src/main/java/hu/bme/mit/theta/xcfa/passes/XcfaPassManager.java

```diff
@@ -46,6 +46,7 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Set;
```

Comment:
Is this necessary? This is the only change in this file.

Reply:
Removed. (It's a result of a debugging session where the XCFA was printed out. Sets were needed them; then I forgot to remove the import.)